pkg/operations/kubernetesupgrade/upgradeagentnode.go

```diff
@@ -54,13 +55,15 @@ func (kan *UpgradeAgentNode) DeleteNode(vmName *string, drain bool) error {
 		kubeAPIServerURL = kan.UpgradeContainerService.Properties.MasterProfile.FQDN
 	}
 
+	nodeName := strings.ToLower(*vmName)
```

Comment:
Just curious: are we validating that `*vmName` isn't nil anywhere in this flow, e.g., before it's passed into this method?

Reply:
It _should_ never be nil since it comes directly from the list of VMs returned by the client but I've added an extra nil check just in case.